comp/dogstatsd/server/enrich_serverless_test.go

```diff
@@ -35,3 +36,90 @@ func TestConvertParseDistributionServerless(t *testing.T) {
 	// an empty string and the parser / enricher should keep the host that way.
 	assert.Equal(t, "", parsed.Host, "In serverless mode, the hostname should be an empty string")
 }
+
+func TestGetServerlessMetricSource(t *testing.T) {
```

Comment:
Are these tags actually relevant anymore now that the logic isn't tags-based?

Reply:
Aha you're right, the tags aren't necessary 🧼